posts/templates/posts/post_detail.html

```diff
@@ -3,7 +3,16 @@
 <div class="container">
   <h1>{{ object.title }}</h1>
   <h3>By {{ object.author }}</h3>
-  <p>{{ object.content }}</p>
+
+  {% if object.content_type == 'text/plain' %}
+    <p>{{ object.content }}</p>
+  {% elif object.content_type == 'image/png;base64' or object.content_type == 'image/jpeg;base64' %}
+      {% if object.imgContent %}
+        <img src="{{ object.imgContent.url }}" height="auto" width="100%" alt="alt"/>
+      {% else %}
+          <img src="{{ post.content }}" height="auto" width="100%" alt="alt"/>
```

Comment:
Where is `post` coming from?

Reply:
Definitely a typo, believe it should say `object`